components/Nav/Navbar/AccountDropdown/AccountDropdown.tsx

```diff
@@ -52,7 +53,7 @@ const AccountDropdownButton = ({ account, ensName, logout }: AccountDropdownButt
                         onClick={() => {
                             /* This requires a secure origin, either HTTPS or localhost. */
                             try {
-                                navigator.clipboard.writeText(account);
+                                navigator.clipboard.writeText(ethers.utils.getAddress(account));
```

Comment:
can `account` ever be an empty string here? I know that `ethers.utils.getAddress` with throw if you pass an empty string

Reply:
nah handled by https://github.com/tracer-protocol/pools-client/blob/1dbbf2b813f3cb364cb7b594359ff7a1af21dee3/components/Nav/Navbar/AccountDropdown/AccountDropdown.tsx#L22-L29